packages/react/src/components/Grid/index.ts

```diff
@@ -5,9 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import { type GridProps } from './GridTypes';
+export { type GridProps };
```

Comment:
Could this be one line?

```suggestion
export { type GridProps } from './GridTypes';
```